src/System.Runtime/tests/System/GCTests.cs

```diff
@@ -112,12 +119,17 @@ public static void KeepAlive()
 
         private class KeepAliveTest
         {
+            [System.Runtime.CompilerServices.MethodImplAttribute(System.Runtime.CompilerServices.MethodImplOptions.NoInlining)]
+            private static void MakeAndDropDNKA()
+            {
+                new DoNotKeepAliveObject();
```

Comment:
I sort of changed my mind as I was editing the various tests, should have gone back and made things more consistent - I'll clean it up a bit and include the changes with my next update. Thanks for catching this.